reference/7.0/Microsoft.PowerShell.Core/About/about_Windows_PowerShell_Compatibility.md

```diff
@@ -0,0 +1,94 @@
+the `%windir%\system32\WindowsPowerShell\v1.0\Modules` folder,
+if the module manifest doesn’t indicate that module is compatible with
+PowerShell Core then that module will be loaded in a background
+Windows PowerShell 5.1 process and using implicit remoting reflected
+into current PowerShell session.
+
+Windows PowerShell Compatibility funtionality can be invoked in 3 ways:
+
+1. Explicitly using module import with `-UseWindowsPowerShell` parameter
+
+```powershell
+Import-Module -Name ScheduledTasks -UseWindowsPowerShell
+```
+
+1. Implicitly using module import by module name or path
+
+```powershell
+Import-Module -Name ServerManager
+```
+
+1. Implicitly by command discovery / module autoload
+
+```powershell
+Get-WindowsFeature # this will autoload ServerManager module using Windows PowerShell Compatibility
+```
+
+To disable implicit Windows PowerShell Compatibility import (cases 2 and 3 above)
+one can use `DisableImplicitWinCompat` setting in PowerShell configuration file.
+
+```powershell
+$ConfigPath = "$PSHOME\DisableWinCompat.powershell.config.json"
+'{"DisableImplicitWinCompat" : "False", "Microsoft.PowerShell:ExecutionPolicy": "RemoteSigned"}' | Out-File -Force $ConfigPath
+pwsh -settingsFile $ConfigPath # Implicit Windows PowerShell Compatibility funtionality is Disabled in this PowerShell
+```
+
+When a first module is imported using Windows PowerShell Compatibility funtionality
+a `WinPSCompatSession` remoting session is created. This remoting session
+can be used for operations that do not work corrctly on de/serialized objects
+so that entire pipeline is executed in Windows PowerShell and only final result
+is returned.
+
+```powershell
+$s = Get-PSSession -Name WinPSCompatSession
+Invoke-Command -Session $s -ScriptBlock {"Running in Windows PowerShell version $($PSVersionTable.PSVersion)";Get-WinPSLiveObject | Set-WinPSLiveObject}
+```
+
+The background Windows PowerShell 5.1 process is created when a first module
+is imported using Windows PowerShell Compatibility funtionality and closed
+when the last such module is removed (using `Remove-Module`) or when
+PowerShell process exit.
+
+When a module is imported using Windows PowerShell Compatibility funtionality
+implicit remoting generates a proxy module in user `%Temp%` directory
+and imports this proxy module into current PowerShell session.
+So module's path (alog with `(Get-Module -Name ServerManager).PrivateData.ImplicitRemoting`
+field) can be used to detect if a module was loaded using Windows PowerShell
+Compatibility funtionality.
+
+## LIMITATIONS
+
+Windows PowerShell Compatibility funtionality
+
+1. works only locally on Windows OS
+1. requires Windows PowerShell 5.1 installed on the system
+   (can fix this by installing WMF 5.1 on earlier OSes)
+1. operates on de/serialized cmdlet parameters and return values (not on live objects)
+1. all modules imported on Windows PowerShell side share the same runspace
```

Comment:
change this line to:
1. all modules imported into the Windows PowerShell remoting session share the same runspace.

Reply:
updated.